classes/class-woothemes-sensei-certificates.php

```diff
@@ -1170,4 +1193,36 @@ public function certificates_user_settings_messages() {
 
 	} // End certificates_user_settings_message()
 
+	public function email_certificate_link( $template ) {
```

Comment:
Also missing a doc block here.

Reply:
Good catch. Added in https://github.com/woocommerce/sensei-certificates/pull/174/commits/746e4d2f7b532567806a61dc0513da4d0264e123